install/0000_90_cluster-version-operator_00_servicemonitor.yaml

```diff
@@ -16,3 +16,42 @@ spec:
   selector:
     matchLabels:
       k8s-app: cluster-version-operator
+---
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    k8s-app: cluster-version-operator
+  name: cluster-version-operator
+  namespace: openshift-cluster-version
+spec:
+  groups:
+  - name: cluster-version
+    rules:
+    - alert: ClusterVersionOperatorDown
+      annotations:
+        message: ClusterVersionOperator has disappeared from Prometheus target discovery.
+      expr: |
+        absent(up{job="cluster-version-operator"} == 1)
+      for: 15m
+      labels:
+        severity: critical
+  - name: cluster-operators
+    rules:
+    - alert: ClusterOperatorDown
+      annotations:
+        message: ClusterOperator {{ $labels.name}} has not been available for 10 mins.
+      expr: |
+        cluster_operator_up{job="cluster-version-operator"}
+      for: 10m
+      labels:
+        severity: critical
+    - alert: ClusterOperatorFlapping
+      annotations:
+        message: ClusterOperator {{ $labels.name}} changing it's up status often
```

Comment:
nit: "it's" -> "its"

Reply:
@wking done with https://github.com/openshift/cluster-version-operator/pull/232/commits/136fb720be157e62cf63b560e65304898c40d989